framework/packages/abstract-interface/src/migrate.rs

```diff
@@ -323,7 +324,13 @@ impl<Chain: CwEnv> AbstractIbc<Chain> {
                 .expect("IBC host supposed to be migrated, but skipped instead");
         } else {
             // Version change is breaking, need to deploy new version
```

Comment:
I've actually come around on this issue, I don't think this solution works will with our workflow. When we release a breaking version of our contracts that doesn't always mean we break the API of the contract. Also, the fact that this contract stores the counterparty contract means that when "upgrading" users lose their account on the other chain. 

Could you think of any way for us to improve this? 

I'm thinking we could stabilize the API and we publish breaking versions under a different module-id, but maybe that's weird as well?

Reply:
> I've actually come around on this issue, I don't think this solution works will with our workflow. When we release a breaking version of our contracts that doesn't always mean we break the API of the contract. Also, the fact that this contract stores the counterparty contract means that when "upgrading" users lose their account on the other chain.

That's why we shouldn't make breaking bump to ibc modules without breaking change!

> Could you think of any way for us to improve this?

Pretty sure if proxy contract will own data about remote accounts then update will become much easier